server/src/database/queries/parcelsQueries/changeStatusQuery.ts

```diff
@@ -0,0 +1,7 @@
+import connection from '../../config/connection';
+
+const changeStatusQuery = ({ sellerId, ParcelId }) => connection.query({
```

Comment:
why you are using pascal convention here, we use camelCase to naming our variables, 
instead of ParcelId must be parcelId